build.gradle

```diff
@@ -39,25 +30,32 @@ repositories {
     maven { url "https://maven.google.com" }
 }
 
+apply plugin: "groovy"
```

Comment:
I just think that if we are applying plugins to the `classpath`, we should we use `apply plugin` after the fact.